### PR TITLE
Added methods of get() and count() to both storage engines

### PR DESCRIPTION
**Pull request by MarcoANT9**

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -55,6 +55,38 @@ def new(self, obj):
...
+                    → If cls == None, returns the count of all objects.
+        """
+        count_classes = 0
+        if cls:
+            objects_in_class = self.all(cls)
+            return (len(objects_in_class))
+        else:
+            for current_class in classes:
+                clss = self.all()
+                count_classes += len(clss)
```
Comment (Jdpa357) [inline models/engine/db_storage.py:87]: Here in count_classes += len(**clss**) doesn't exist or i dont see it in the file, if it contains any instantiation of this variable i dont see it anywhere. please correct it

**Review by Jdpa357 — CHANGES_REQUESTED** [review]
Lots of pycache unnecesary files in the commit, remove them from the commit before merging into the project.
Also the db_storage.py has an error addressed on line 87. look at the comments i left there

File: models/engine/db_storage.py (hunk 1 above)
Comment (MarcoANT9) [reply models/engine/db_storage.py:87]: Thats on the return of that part of the code, its definition is at the beggining of that method, I'll move it to that section.

Comment (Jdpa357) [reply models/engine/db_storage.py:87]: oh i got it, lol i don't know how i pass it over

**Review by Jdpa357 — APPROVED** [review]

**Comment by Jdpa357** [comment]
Changes done on temporary cache files. All good, tests done

**Pull request merged**